lib/core/versions/0.4.0/TransactionSelector.ts

```diff
@@ -0,0 +1,15 @@
+import ITransactionSelector from '../../interfaces/ITransactionSelector';
+import TransactionModel from '../../../common/models/TransactionModel';
+
+/**
+ * dummy transaction selector
```

Comment:
is this accurate documentation?

Reply:
Yes, they are not supposed to do anything but return what you pass in.